gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-rc-2-bin.zip
```

Comment:
In fact, I think we should not make this change in this PR, This just affects the Gradle version that is used to *build* the plugin. https://github.com/ben-manes/gradle-versions-plugin/pull/131/commits/ba4d411e955a4dff40a40c05c78585fed82fc515 adds 3.4-rc-2 to the versions to test against.